.slack/apps.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
I don't think we want this checked in, do we?

Reply:
@misscoded If that’s the case, we should change .gitignore for all templates. Should we?

Reply:
Not sure about the reason why we currently ignore only dev json files

Reply:
I swore I had a previous discussion with someone that indicated both shouldn't be checked in, but I defer. It's not a use case I've thought about at all.

Reply:
Thanks for the replies!